src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -101,7 +101,13 @@ private static TestInputConfiguration parse(String inputFilePath,
         final List<String> lines = readFile(filePath);
         final TestInputConfiguration.Builder testInputConfigBuilder =
                 new TestInputConfiguration.Builder();
-        setModules(testInputConfigBuilder, inputFilePath, lines);
+        try {
+            setModules(testInputConfigBuilder, inputFilePath, lines);
+        }
+        catch (CheckstyleException ex) {
+            throw new CheckstyleException("Config comment not specified properly in "
```

Comment:
please catch all exceptions (`catch (Exception exc)`) at this level and print that there are some problems. Please put whole code of this method in such try-catch as exception can appear at any place.
There will be bunch of violations from checkstyle and pmd of this, we can suppress them, because it is the most top place for parsing and we need to wrap at this level to print what we tried to parse. 

Some inner/caused-by exception will have more details on what exactly is wrong.